pylint/checkers/classes.py

```diff
@@ -133,6 +133,26 @@ def _positional_parameters(method):
         positional = positional[1:]
     return positional
 
+def _has_different_parameters_default_value(original, overridden):
+    """
+    Return True if one of the overridden arguments has a default
+    value different from the default value of the original argument
+    """
+    orig_name = [param.name for param in original.args]
```

Comment:
`.as_string()` is not kind of API you want to use in this case.

Maybe consider using standard sentinel notion. Something around the lines of:
 
```python
default_missing = object()
for param_name in orig_name:
  try:
    orig_default = original.default_value(param_name)
  except astroid.exceptions.NoDefault:
    orig_default = default_missing
  try:
    over_default = overridden.default_value(param_name)
  except astroid.exceptions.NoDefault:
    over_default = default_missing
  if orig_default != over_default
    return True
return False
```